tutorials/telco/charmed-osm-get-started/charmed-osm-get-started.md

```diff
@@ -0,0 +1,840 @@
+```
+
+## Setting up MicroStack
+
+Duration: 10:00
+
+To install MicroK8s execute:
+
+```bash
+$ sudo snap install microstack --classic --beta
+microstack (beta) stein from Canonical✓ installed
+```
+
+Once completed, execute the following commands:
+
+```bash
+$ sudo microstack.init --auto
+2019-11-10 20:54:14,841 - microstack_init - INFO - Setting up ipv4 forwarding...
+2019-11-10 20:54:14,940 - microstack_init - INFO - Waiting for RabbitMQ to start ...
+Waiting for 10.20.20.1:5672
+2019-11-10 20:54:14,949 - microstack_init - INFO - RabbitMQ started!
+2019-11-10 20:54:14,949 - microstack_init - INFO - Configuring RabbitMQ ...
+2019-11-10 20:54:16,766 - microstack_init - INFO - RabbitMQ Configured!
+2019-11-10 20:54:16,798 - microstack_init - INFO - Waiting for MySQL server to start ...
+Waiting for 10.20.20.1:3306
+2019-11-10 20:54:16,803 - microstack_init - INFO - Mysql server started! Creating databases ...
+/snap/microstack/171/lib/python3.6/site-packages/pymysql/cursors.py:170: Warning: (1287, 'Using GRANT for creating new user is deprecated and will be removed in future release. Create new user with CREATE USER statement.')
+  result = self._query(query)
+2019-11-10 20:54:17,791 - microstack_init - INFO - Configuring Keystone Fernet Keys ...
+2019-11-10 20:54:34,892 - microstack_init - INFO - Bootstrapping Keystone ...
+2019-11-10 20:54:40,374 - microstack_init - INFO - Creating service project ...
+2019-11-10 20:54:45,931 - microstack_init - INFO - Keystone configured!
+2019-11-10 20:54:45,967 - microstack_init - INFO - Configuring nova ...
+/snap/microstack/171/lib/python3.6/site-packages/pymysql/cursors.py:170: Warning: (1287, "Using GRANT statement to modify existing user's properties other than privileges is deprecated and will be removed in future release. Use ALTER USER statement for this operation.")
+  result = self._query(query)
+Waiting for 10.20.20.1:8774
+2019-11-10 20:56:26,974 - microstack_init - INFO - Creating default flavors...
+2019-11-10 20:56:52,934 - microstack_init - INFO - Configuring Neutron
+Waiting for 10.20.20.1:9696
+2019-11-10 20:58:15,223 - microstack_init - INFO - Configuring Glance ...
+Waiting for 10.20.20.1:9292
+2019-11-10 20:58:50,777 - microstack_init - INFO - Fetching cirros image ...
+100% [........................................................................] 12716032 / 127160322019-11-10 20:58:57,486 - microstack_init - INFO - Creating microstack keypair (~/.ssh/id_microstack)
+2019-11-10 20:59:00,262 - microstack_init - INFO - Creating security group rules ...
+2019-11-10 20:59:10,779 - microstack_init - INFO - restarting libvirt and virtlogd ...
+2019-11-10 20:59:10,928 - microstack_init - INFO - Complete. Marked microstack as initialized!
+```
+
+That’s it! Your local OpenStack cluster is up and running. You can interact with it by using the `microstack.openstack` command.
+
+In order to configure MicroStack, so that it could be used as a VIM for Charmed OSM, add Ubuntu image:
```

Comment:
```suggestion
In order to configure MicroStack, so that it could be used as a VIM for Charmed OSM, you need to add an Ubuntu image:

Download Ubuntu Server 16.04 LTS with the following command:
```